sample/applicationinsights-react-sample/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "applicationinsights-react-sample",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@microsoft/applicationinsights-react-js": "^3.3.6",
```

Comment:
Can we wire it up with rush?
Add this project to the `/rush.json` and then than this reference so it's the "current" version.

This will cause this project to get compiled using the current code (rather than fetching the "latest" version from npm) so that if we make some sort of breaking change this example would fail.

Reply:
I "think" the setVersion script (in `/release-tools/setVersion.js`) would then also automatically update this package.json reference for us as well -- so as long as we use that script for a release we shouldn't end up with this out of sync.

Reply:
The internal build is failing because the `function shouldProcess(name) {` portion of the setVersion.js (in release-tools) is not updating the new samples folder.

And the PR build runs setVersion causing the build number to be appended with a -dev tag, which is why it's missing.

If the setVersion updated the  version and you have line 8 set to the same version number -- things should work.